packages/jsii/test/test.enums.ts

```diff
@@ -38,6 +39,7 @@ export = {
       fqn: 'testpkg.Foo',
       kind: 'enum',
       members: [{ name: 'Bar' }, { name: 'Baz' }],
```

Comment:
Would be nice if the members were also locatable.... 😇 (But ... like not important at all).